contracts/SportMarkets/Rundown/TherundownConsumer.sol

```diff
@@ -615,10 +627,15 @@ contract TherundownConsumer is Initializable, ProxyOwned, ProxyPausable {
         address _market,
         uint _outcome,
         uint8 _homeScore,
-        uint8 _awayScore
+        uint8 _awayScore,
+        uint8 _status
     ) internal {
         sportsManager.resolveMarket(_market, _outcome);
-        oddsObtainer.resolveChildMarkets(_market, _outcome, _homeScore, _awayScore);
+        if (_status == 201) {
+            oddsObtainer.resolveChildMarkets(_market, 0, 0, 0);
```

Comment:
I was considering to check if Totals can be resolved as over already based on the current result. e.g. if result sent is 2:1 and totals threshold was 3, perhaps we can resolve the totals markets as OVER.

Reply:
But not sure tbh about this